builtin/settingtypes.txt

```diff
@@ -384,6 +393,11 @@ enable_clouds (Clouds) bool true
 #    Requires: enable_clouds
 enable_3d_clouds (3D clouds) bool true
 
+#   Use smooth cloud shading.
+#
+#   Requires: enable_3d_clouds, enable_clouds
```

Comment:
inferring from the screenshots this just seems to be a change of the color of the top side of clouds, right?
that should definitely just be up to the API user

Reply:
No, this changes the blending. Normally, all faces are shaded as "flat" and this makes a smooth color gradient from top to bottom. Cloud color would be API controlled. I don't see why this should be controlled by API though, given that it just changes the shading.

Reply:
I agree this does not need to be API controlled.